src/lib/src/select-container.component.ts

```diff
@@ -225,7 +222,7 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
     // Update the container as well as all selectable items if the list has changed
     this.$selectableItems.changes.pipe(takeUntil(this.destroy$)).subscribe((items: QueryList<SelectItemDirective>) => {
       const newList = items.toArray();
-      const removedItems = this._selectedItems.filter(item => !newList.includes(item.value));
+      const removedItems = this._selectedItems$.value.filter(item => !newList.includes(item.value));
```

Comment:
I am not a big fan of accessing the last value via the `value` property. To me this feels like an anti-pattern and not really reactive. Here, we could instead use `withLatestFrom`. Seems to be a bit more code to write and it also adds another operator but it feels more correct to me. This may also be subjective but personally I try to avoid this as much as possible.